### PR TITLE
Issue58

### PR DESCRIPTION
**Pull request by glandre**
close issue #58

### CODE REVIEW

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java [hunk 1]
```diff
@@ -0,0 +1,240 @@
+package latexstudio.editor;
```
Comment (sebbrudzinski) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: Can we move it to a new package, to distinguish that it is an external solution? The package could be named lib.

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+ *  source of the Action will be the table. The action command will contain
+ *  the model row number of the button that was clicked.
+ * 
+ * @author Rob Camick
+ * @see https://tips4java.wordpress.com/2009/07/12/table-button-column/
+ *
+ */
+public class ButtonColumn extends AbstractCellEditor
+	implements TableCellRenderer, TableCellEditor, ActionListener, MouseListener
+{
```
Comment (sebbrudzinski) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: I know this is someone else's code, but can we adjust the braces to match project (and Java!) coding standards? More specifically, the opening brace should be in the same line as method signature.

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    {
+    	if (isButtonColumnEditor
+    	&&  table.isEditing())
+    		table.getCellEditor().stopCellEditing();
+
+		isButtonColumnEditor = false;
+    }
+
+    public void mouseClicked(MouseEvent e) {}
+	public void mouseEntered(MouseEvent e) {}
```
Comment (sebbrudzinski) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: Unnecessary tab

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/DropboxRevisionsTopComponent.java [hunk 4]
```diff
@@ -54,17 +66,20 @@
...
+
     private static final RevisionDisplayTopComponent REVTC = new TopComponentFactory<RevisionDisplayTopComponent>()
             .getTopComponent(RevisionDisplayTopComponent.class.getSimpleName());
-    
+
     private static final String REVISION_COLUMN_NAME = "Revision";
     private static final String MODIFIED_COLUMN_NAME = "Modified";
     private static final String FILE_SIZE_COLUMN_NAME = "File size";
+    private static final String REVIEW_COLUMN_NAME = "Review";
+    private static final String REVIEW_BUTTON_LABEL = "view revision";
```
Comment (sebbrudzinski) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/DropboxRevisionsTopComponent.java]: Capitalize - "View Revision"

**Comment by sebbrudzinski** [comment]
@glandre It works, but I've got some comments. Also, can we expand the height of the rows a little? Currently the text on the button is barely visible.

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java (hunk 1 above)
Comment (glandre) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: Only lib right? Or you mean latexstudio.lib?

Comment (sebbrudzinski) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: I meant latexstudio.lib

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java (hunk 2 above)
Comment (glandre) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: OK, done.

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java (hunk 3 above)
Comment (glandre) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: Adjusted. I also edited the comments and annotations.

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/DropboxRevisionsTopComponent.java (hunk 4 above)
Comment (glandre) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/DropboxRevisionsTopComponent.java]: Done.

**Comment by glandre** [comment]
Hello @sebbrudzinski, please check it out.

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java (hunk 1 above)
Comment (glandre) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/ButtonColumn.java]: OK. Done.

**Pull request merged**